test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -134,11 +136,13 @@ func schedulePods(config *testConfig) int32 {
 	// We are interested in low scheduling rates (i.e. qps=2),
 	minQPS := int32(math.MaxInt32)
 	start := time.Now()
-	podLister := config.schedulerSupport.PodInformer.Lister()
+
+	informers := informers.NewSharedInformerFactory(config.clientset, 0)
```

Comment:
@Huang-Wei thanks for the review, I created a new pod informer with clientset here, WDYT?